packages/color-block/src/types.ts

```diff
@@ -0,0 +1,80 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { ReactNode } from 'react';
+
+import { AppBridgeBlock, FrontifyColor, FrontifyColorPatch } from '@frontify/app-bridge';
+import { Color } from '@frontify/fondue';
+
+export type ColorBlockProps = {
+    appBridge: AppBridgeBlock;
+};
+
+export type Settings = {
+    name?: string;
+    description?: string;
+    view?: ColorBlockType;
+    colorspaces?: string[];
```

Comment:
```suggestion
    colorspaces?: (keyof ColorSpaceInputValues)[];
```

Reply:
This comes from BlockSettings, not sure how can I redefine it